grakn-dist/src/main/java/ai/grakn/dist/DistGraql.java

```diff
@@ -0,0 +1,118 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.dist;
+
+import ai.grakn.GraknSystemProperty;
+import ai.grakn.graql.GraqlShell;
+import ai.grakn.migration.csv.CSVMigrator;
+import ai.grakn.migration.json.JsonMigrator;
+import ai.grakn.migration.sql.SQLMigrator;
+import ai.grakn.migration.xml.XmlMigrator;
+
+import java.util.Arrays;
+
+/**
+ *
+ * @author Michele Orsi
+ */
+public class DistGraql {
+
+    /**
+     * In order to run this method you should have 'grakn.dir' set
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        String home = GraknSystemProperty.CURRENT_DIRECTORY.value();
+
+        if(home==null) {
+            System.out.println("Problem with bash script: cannot run Grakn");
+            return;
+        }
+
+        DistGraql application = new DistGraql();
+        try {
+            application.run(args);
+        } catch (RuntimeException ex) {
+            System.out.println(ex.getMessage());
+        }
+
+    }
+
+    private void run(String[] args) {
+        String arg0 = args.length > 0 ? args[0] : "";
+
+        switch (arg0) {
+            case "console":
+                GraqlShell.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "migrate":
+                migrate(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "version":
+                version();
+                break;
+            default: help();
+        }
+
+    }
+
+    private void migrate(String[] args) {
+        String arg0 = args.length > 0 ? args[0] : "";
+
+        switch (arg0) {
+            case "csv":
+                CSVMigrator.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "json":
+                JsonMigrator.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "owl":
+                System.out.println("Owm migration not supported anymore");
+                break;
+            case "export":
+                ai.grakn.migration.export.Main.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "sql":
+                SQLMigrator.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            case "xml":
+                XmlMigrator.main(Arrays.copyOfRange(args, 1, args.length));
+                break;
+            default:
+                XmlMigrator.main(new String[]{"-h"});
```

Comment:
Shouldn't this just print the regular `help` below?

Reply:
I think you're right: it was my mistake of just copying from bash scripting ..